src/components/TaskListScreen.js

```diff
@@ -0,0 +1,134 @@
+import "./TaskListScreen.scss";
+import firebase from "firebase";
+import { useState, useEffect } from 'react';
+import {
+    Switch,
+    Route,
+    Link,
+    Redirect,
+    useHistory
+} from "react-router-dom";
+import { useSelector } from "react-redux";
+import addIcon from "../svg/add-icon.svg";
+import TextAvatar from "./TextAvatar";
+import TaskScreen from "./TaskScreen";
+import moment from 'moment';
+import 'moment/locale/ru';
+
+function TaskListScreen() {
+    const profile = useSelector(state => state.profile);
+    const [newTaskName, setNewTaskName] = useState("");
+    const [newTaskDescription, setNewTaskDescription] = useState("");
+    const [tasks, setTasks] = useState([]);
+    const history = useHistory();
+    moment.locale('ru')
+
+    useEffect(() => {
+        if (!profile)
+            return;
+
+        var query = firebase.firestore().collection("tasks");
```

Comment:
Using `var` variables inside the `function component` is not the best practice. Use `let` or `const`.